kafka-ui-react-app/src/redux/actions/thunks/connectors.ts

```diff
@@ -196,13 +196,44 @@ export const restartConnector =
       const response = await getResponse(error);
       const alert: FailurePayload = {
         subject: [clusterName, connectName, connectorName].join('-'),
-        title: `Kafka Connect Connector Tasks Restart`,
+        title: `Kafka Connect Connector Restart`,
         response,
       };
       dispatch(actions.restartConnectorAction.failure({ alert }));
     }
   };
 
+export const restartTasks =
+  (
+    clusterName: ClusterName,
+    connectName: ConnectName,
+    connectorName: ConnectorName,
+    action: ConnectorAction
+  ): PromiseThunkResult<void> =>
+  async (dispatch) => {
+    dispatch(actions.restartTasksAction.request());
+    try {
+      await kafkaConnectApiClient.updateConnectorState({
+        clusterName,
+        connectName,
+        connectorName,
+        action,
+      });
+      dispatch(actions.restartTasksAction.success());
```

Comment:
I think since we have multiple dispatches here we can batch those , with the new redux Api we actually have this feature to decrease re-rendering